scripts/onboarding/templates/arc-k8s-extension-msi-auth/existingClusterOnboarding.json

```diff
@@ -128,6 +128,11 @@
     "dceName":"[if(greater(length(variables('dcrName')), 43), substring(variables('dcrName'), 0, 43), variables('dcrName'))]",
     "dceAssociationName": "configurationAccessEndpoint",

```

Comment:
@wanlonghenry , please list out the scenarios which are validated.

Reply:
tested both high scale and non high scale mode please check in describe